nibabel/tests/test_scripts.py

```diff
@@ -54,15 +54,62 @@ def script_test(func):
 
 
 @script_test
-def test_nib_ls():
+def _test_nib_ls_example4d(opts=[], hdrs_str=""):
```

Comment:
Elsewhere I've been using `check_` prefix for things that do asserts, that aren't meant to be tests directly.

Does it need the decorator?